CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [2.1.1] - 2021-04-21
```

Comment:
This fix is unreleased yet, so the `### Added` must be under [Unreleased]. I add version number when it'll be tagged.

Reply:
Fixed